### PR TITLE
Feature/tools to community

### PR DESCRIPTION
**Pull request by brad-allen**
Moved the former MistyRobotics.Tools, now SkillTools, nuget helper code into community repo.

### CODE REVIEW

File: C#/Libraries/SkillTools/SkillTools/Web/WebMessengerData.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		/// <summary>
+		/// Http response code
+		/// </summary>
+		public int HttpCode { get; set; }
+
+		/// <summary>
+		/// Response as text
+		/// </summary>
+		public string Response { get; set; }
+
```
Comment (adam-misty) [inline C#/Libraries/SkillTools/SkillTools/Web/WebMessengerData.cs:17]: Bonus line

File: C#/Libraries/SkillTools/SkillTools/DataStorage/BasicStorage.cs [hunk 2]
```diff
@@ -0,0 +1,213 @@
+﻿using System;
```
Comment (adam-misty) [inline C#/Libraries/SkillTools/SkillTools/DataStorage/BasicStorage.cs:1]: Should all of these code files have the license blurb?

File: C#/Libraries/SkillTools/SkillTools/Web/WebMessengerData.cs (hunk 1 above)
Comment (brad-allen) [reply C#/Libraries/SkillTools/SkillTools/Web/WebMessengerData.cs:17]: woot!  congrats :D

File: C#/Libraries/SkillTools/SkillTools/DataStorage/BasicStorage.cs (hunk 2 above)
Comment (brad-allen) [reply C#/Libraries/SkillTools/SkillTools/DataStorage/BasicStorage.cs:1]: Adding a license file

**Review by johnathanDOS — APPROVED** [review]

**Pull request merged**